### ISSUE TITLE
IScroll and Phonegap on Android 5

### ISSUE THREAD

**Issue by mcbeth111** [cubiq/iscroll#840]
Anyone made it working on Lollipop ? I have a Phonegap app working fine on previous Androids and iOS but on Android 5 wrapper content does not scroll or zoom. The same thing with iScroll 4. What's wrong ?

**Comment by andregu** [comment]
i have it working on android L and i didnt change anything.

**Comment by ms88privat** [comment]
works for me too, nothing changed. But I do not use zoom

**Comment by emtopping** [comment]
Using iScroll 4 scroll works but zoom does not on Lollipop

**Comment by mcbeth111** [comment]
You are right. The zoom is the problem on Android 5. Do you have any good alternative to iscroll working on Lollipop ?

**Comment by emtopping** [comment]
@mcbeth111 I used iScroll-Zoom.js from V5 just for the Zoom bit (as I din't want to re-test everything else/rewrite scroll code). The name change means there is no conflict...

//Use V5 of iScroll ONLY for Zooming
    var mapScroll;
    try
    {
        console.log(device.platform);
        console.log(device.version);
        mapScroll = new IScroll('#map-wrapper', {
            zoom: true, scrollX: true,
            scrollY: true,
            click: true
        });
    }

**Comment by MerlinMags** [comment]
I have had to drop iScroll from my Phonegap app as it does not work on Android v5.1.1 or v7.1.1 (despite still working beautifully on iOS). Zooming with pinch/expand gesture does nothing. Slow or fast swipes do not move the element around smoothly; it just makes a small jerk in the right direction and then stops. Should I blame my own code for this, or blame Android?!
